src/shared/meson.build

```diff
@@ -466,6 +466,7 @@ libshared_deps = [threads,
                   libselinux,
                   libzstd,
                   libxz,
+                  libcryptsetup,
```

Comment:
this is wrong. we should not link against libcryptsetup directly. We load it only via dlopen().

it's OK to add the libcryptsetup includes to the includes list, but full-on linking is not ok.

Reply:
this line still needs to be removed

Reply:
sorry, done now